tests/python/mints14/input.py

```diff
@@ -71,3 +71,142 @@
+                 [ 1.63796773e-03,  4.85361378e-04, -6.16196031e-04],
+                 [ 1.29367916e-03,  5.25212195e-04,  1.03863118e-04],
+                 [ 8.59327260e-04,  4.01031500e-04,  3.59125641e-04],
+                 [ 2.93944897e-04, -3.51934887e-04, -1.34870686e-03],
+                 [ 8.90411452e-04,  2.64034693e-04, -1.05652614e-03],
+                 [ 1.04656838e-03,  5.75305304e-04, -4.47404228e-04],
+                 [ 8.70186444e-04,  5.74702439e-04,  1.00362330e-05],
+                 [ 6.20294372e-04,  4.51547988e-04,  2.17294907e-04],
+                 [ 2.37081931e-04, -7.24049767e-05, -8.31791663e-04],
+                 [ 5.03400058e-04,  2.91236028e-04, -6.66530072e-04],
+                 [ 5.93333149e-04,  4.83714963e-04, -3.42938122e-04],
+                 [ 5.31104808e-04,  4.93654625e-04, -6.25231120e-05],
+                 [ 4.11842208e-04,  4.12457848e-04,  9.85127268e-05],
+                 [-8.42819831e-04, -1.08420217e-19, -1.49460411e-03],
+                 [ 2.79295542e-04,  2.16840434e-19, -1.41942351e-03],
+                 [ 9.38038156e-04,  0.00000000e+00, -7.81227007e-04],
+                 [ 9.92258235e-04,  1.08420217e-19, -1.87317941e-04],
+                 [ 7.81410855e-04,  1.08420217e-19,  1.27287507e-04],
+                 [-5.97125829e-04, -3.79755298e-04, -1.37910616e-03],
+                 [ 3.63061171e-04, -1.31021975e-04, -1.28521300e-03],
+                 [ 9.08362134e-04,  6.07419159e-05, -7.10817802e-04],
+                 [ 9.37847191e-04,  1.30354714e-04, -1.74719261e-04],
+                 [ 7.39869491e-04,  1.25749968e-04,  1.16183350e-04],
+                 [-1.82953325e-04, -4.09294959e-04, -1.10541241e-03],
+                 [ 4.52362270e-04, -8.18711829e-05, -9.92315897e-04],
+                 [ 7.89766516e-04,  1.55470402e-04, -5.61407932e-04],
+                 [ 7.87393944e-04,  2.36094511e-04, -1.52733690e-04],
+                 [ 6.29223855e-04,  2.20222896e-04,  8.47326157e-05],
+                 [ 4.44809882e-05, -2.48424922e-04, -8.06800681e-04],
+                 [ 4.07558302e-04,  3.06044162e-05, -7.05944729e-04],
+                 [ 5.94617785e-04,  2.22564547e-04, -4.20269365e-04],
+                 [ 5.89143701e-04,  2.84872327e-04, -1.38704841e-04],
+                 [ 4.85732587e-04,  2.63559971e-04,  4.21765372e-05],
+                 [ 9.79431484e-05, -1.09066464e-04, -5.63407299e-04],
+                 [ 2.94543076e-04,  9.13557097e-05, -4.90653651e-04],
+                 [ 3.99793079e-04,  2.27763976e-04, -3.13890974e-04],
+                 [ 4.03799551e-04,  2.74855411e-04, -1.29722537e-04],
+                 [ 3.48015799e-04,  2.59140058e-04,  2.65049640e-06],
+                 ]
+
+# Comparison
+psi4.compare_arrays(efs_1threads, efs_4threads, 3, "Reference value for EF calculation, 1 thread vs. 4 threads.")
+psi4.compare_arrays(efs_1threads, reference_efs, 3, "Reference value for EF calculation, 1 thread vs. reference calculation.")
```

Comment:
Could the test tolerance be tightened up beyond 1.e-3? I worry that much of the reference array is smaller.

Thanks for this new functionality!

Reply:
1e-3 is a pretty significant error for an observable! I would thing this should be accurate to 1e-9 or 1e-10, since the integrals are analytic!

Reply:
Ah good call - I've dropped the tolerance down in 3850718 to 1e-9 and the test still seems to pass ok.